src/dcp/dcp.c

```diff
@@ -332,6 +352,14 @@ int main(int argc, char** argv)
         }
     }
 
+    /* copy_xattrs only vaid with preserve */
+    if (mfu_copy_opts->preserve == false && copy_xattrs_used == 1) {
+        if (rank == 0) {
+            MFU_LOG(MFU_LOG_ERR, "Copy-xattrs may only be used with --preserve");
```

Comment:
IMHO, errors like this are counter productive for users.  I just means if the user _wants_ to copy xattrs, they will hit an error, and then have to fix what the program could have done itself - enable the preserve option if --xattrs is used.